.env.sample

```diff
@@ -1,2 +0,0 @@
-POINTSBOT_TOKEN=xxxxx
```

Comment:
was removing this file intentional?

Reply:
Nope. I had renamed .env.sample to .env. not entirely sure why it got deleted. Will need to fix.

Good catch.

Reply:
There. That should be that screwup fixed.